src/styles/_base.css

```diff
@@ -12,7 +12,10 @@
   margin: 0;
   display: grid;
   grid-template-columns: 1fr min(40ch, calc(100% - 32px)) 1fr;
-  grid-template-rows: 60px auto 80px;
+  grid-template-rows: 48px auto 72px;
+  position: fixed;
+  top: 0;
+  left: 0;
```

Comment:
Line 14 is for wider screens so you have a minimum and a maximum that includes the column 1 and 3 so you have the possibility to use a bleeding edge effect for victory or something. With line 16 to 18 you are right, i tried to solve the problem with the grid gap at the end of the main element what is kinda hack and i am not sure if i really need this lines. But i change back to a div with an white-space utility class because its more clear what the code does.